auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.java

```diff
@@ -1573,6 +1573,47 @@ public void shouldFetchProfileAfterLoginRequest() throws Exception {
         assertThat(callback, hasPayloadOfType(Authentication.class));
     }
 
+
+    @Test
+    public void shouldRevokeToken() throws Exception {
+        Auth0 auth0 = new Auth0(CLIENT_ID, mockAPI.getDomain(), mockAPI.getDomain());
+        AuthenticationAPIClient client = new AuthenticationAPIClient(auth0);
+
+        mockAPI.willReturnSuccessfulEmptyBody();
+        final MockAuthenticationCallback<Void> callback = new MockAuthenticationCallback<>();
+        client.revokeToken("refreshToken")
+                .start(callback);
+
+        final RecordedRequest request = mockAPI.takeRequest();
+        assertThat(request.getHeader("Accept-Language"), is(getDefaultLocale()));
+        assertThat(request.getPath(), equalTo("/oauth/revoke"));
+
+        Map<String, String> body = bodyFromRequest(request);
+        assertThat(body, hasEntry("client_id", CLIENT_ID));
+        assertThat(body, hasEntry("token", "refreshToken"));
+
+        assertThat(callback, hasNoError());
+    }
+
+    @Test
+    public void shouldRevokeTokenSync() throws Exception {
+        Auth0 auth0 = new Auth0(CLIENT_ID, mockAPI.getDomain(), mockAPI.getDomain());
+        auth0.setOIDCConformant(true);
```

Comment:
Disregard my comment.. The oidc flag won't make a change on this endpoint since it should be callable anyway..